plugins/chain_api_plugin/chain.swagger.yaml

```diff
@@ -89,6 +89,75 @@ paths:
             application/json:
               schema:
                 $ref: "https://eosio.github.io/schemata/v2.1/oas/BlockInfo.yaml"
+  /get_finalizer_info_result:
+    post:
+      description: Returns information about finalizers
+      operationId: get_finalizer_info_result
+        responses:
+          "200":
+            description: OK
+            content:
+              application/json:
+                schema:
+                  active_finalizer_policy:
+                    type: object
+                    description: List of current finalizers
+                      generation:
+                        type: integer
+                        description: version number of finalizers set
+                      threshold:
+                        type: integer
+                        description: total weight needed to advance finality
+                      finalizers:
+                        type: array
+                        items:
+                          ref: "#/components/schemas/Finalizer"
+                  pending_finalizer_policy:
+                    type: object
+                    description: List of current finalizers
+                      generation:
+                        type: integer
+                        description: version number of finalizers set
+                      threshold:
+                        type: integer
+                        description: total weight needed to advance finality
+                      finalizers:
+                        type: array
+                        items:
+                          ref: "#/components/schemas/Finalizer"
+                  last_tracked_votes:
+                    type: array
+                    description: Record of last votes for top 21 finalizers
```

Comment:
Not top 21 finalizers. We return the last votes for the finalizers in current active finalizer policy and pending finalizer policy.